### PR TITLE
Leave out entire folder including all subfolders

### PR DESCRIPTION
**Pull request by Kondamon**
Included case study for leaving out an entire folder including all subfolders

### CODE REVIEW

**Comment by Kondamon** [comment]
I have forgotten the not: at the beginning of the line "!". COuld you please add it? Thanks!

File: README.md [hunk 1]
```diff
@@ -194,6 +195,18 @@ end
...
 This block will return true only if the project is **not** contained within a folder that has the name "Template".
 
+### Leaving out an entire folder including all subfolders
+
+Sometimes you want to leave out an entire folder including all subfolders. For example, when your folder "Template" also has subfolders such as "Work Projects", "Private Things" or if you add additional subfolders or rename existing subfolders from time to time. The easiest way to block all subfolders as well:
+
+```ruby
+Omniboard::Column.config do
+	conditions do |p|
+		(p.ancestry.map(&:name).join("→")).include?("Template")
```
Comment (jyruzicka) [inline README.md]: You should be able to do this without `join`ing everything together:

```
conditions do |p|
  !p.ancestry.any?{ |c| c.name == "Template" }
end
```

**Comment by jyruzicka** [comment]
Hey @Kondamon, thanks for contributing! You can modify the file and push it to github again, and it will automatically update this pull request. [For example](http://stackoverflow.com/questions/16748115/how-to-modify-github-pull-request#16748236).

**Comment by Kondamon** [comment]
I hope I did it correct!?

**Comment by jyruzicka** [comment]
Looks good! One last thing you'll want to do is rebase your pull request so your two commits will get merged into one commit - it looks a lot nicer in general. Check out [this article](https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request) under "Squash Changes". You'll know if it works because the "Commits" tab on this pull request will end up with only one commit.

**Comment by Kondamon** [comment]
Thank you for your hints!!

**Comment by jyruzicka** [comment]
👍 

Pull requests are still about 90% mystery to me. Everything looks good!

**Pull request merged**